LED interface code updated

cag343 opened the pull request:
Added GPIO and time libraries, code that turns LEDs on and off, and a stop_all function. Still need to clean up code and fix it

Commit:
LED interface code updated

Added GPIO and time libraries, code that turns LEDs on and off, and a stop_all function. Still need to clean up code and fix it

catvec commented on led_interface.py:
```diff
@@ -7,17 +9,26 @@ class LedInterface():
-    def turn_on_led(self, position):
-        print("Takes a chess coordinate pair (position) to turn on an LED to show initial position")
+    #Sets up the number system for the board and the output pin (location 12 on the diagram)
+    GPIO.setmode(GPIO.BCM)
```
Indent so in __init__ method

catvec commented on led_interface.py:
```diff
@@ -7,17 +9,26 @@ class LedInterface():
+            GPIO.output(12, 1)
+            time.sleep(1)
+            GPIO.output(12, 0)
+            time.sleep
```
Incomplete function call

catvec reviewed (commented):
We will clean this up and make it do things at the meeting, I just noticed 2 things that make it not valid python code.

Commit:
Fixed code

Put setmode and setup in __init__ function and added time to second time.sleep

Commit:
Created fake GPIO and hardware mode

Added hardware mode, fake GPIO file, and began a test file

Commit:
Add files via upload

Copied over the matrix control functions. Still need to fix original 4 functions

Commit:
Add files via upload

Finalized LED code

catvec committed:
Merge branch 'master' into led_blinking_code
